value-resolver/src/main/kotlin/com/ritense/valueresolver/ValueResolverFactory.kt

```diff
@@ -90,4 +91,24 @@ interface ValueResolverFactory {
      * @param values The values to handle. i.e. mapOf(doc:add:/firstname to John)
      */
     fun handleValues(processInstanceId: String, variableScope: VariableScope?, values: Map<String, Any>)
```

Comment:
Maybe add a @Deprecation notice, so we can refactor it in the next major?

Reply:
I think we should do that when we introduce the new way. That way, people don't see the notice without us providing an alternative.